phygnn/layers/custom_layers.py

```diff
@@ -478,8 +478,7 @@ def __init__(self, name):
             Unique string identifier of the skip connection. The skip endpoint
             should have the same name.
         """
-        super().__init__()
-        self._name = name
+        super().__init__(name=name)
```

Comment:
See the error message on line 509 that needs to be udpated

Reply:
the super class initializes a `_name` attribute

Reply:
Ah gotcha, haha so _name was basically the same as name huh? Well all good then. I do think it's better to use .name everywhere then but all good, doesnt matter